e3sm_to_cmip/cmor_handlers/tasmax.py

```diff
@@ -0,0 +1,51 @@
+"""
+TREFMXAV to tasmax converter
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import cmor
+from e3sm_to_cmip.lib import handle_variables
+
+# list of raw variable names needed
+RAW_VARIABLES = [str('TREFMXAV')]
+VAR_NAME = str('tasmax')
+VAR_UNITS = str('K')
+TABLE = str('CMIP6_Amon.json')
```

Comment:
The conversion of string to string using `str()` is probably carried over from another handler file. Not a big deal since the API design is being refactored anyways.

No action needed.

Reply:
Thanks for reviewing! I just copied the codes from other handlers as template. Yes, Let's fix them in the refactor.